### ISSUE TITLE
Add documentation about usage and purpose

### ISSUE THREAD

**Issue by fenio** [evroon/bracket#415]
It's not cleaar from description that this software is designed to handle chess tournaments.
I had to install it to figure out it's not the best fit for soccer tournament ;)

**Comment by evroon** [comment]
Hi, what makes this software unsuitable for soccer? It supports round robin, elimination and ladder tournaments, is there another type of tournament you're looking for?

**Comment by fenio** [comment]
Hm... maybe I didn't spend enough time to find a way to do it.
In fact I need something really simple as this has nothing to do with professional stuff.
Basically we're playing indoor soccer with friends and we split ourselves to 3 teams and play several matches between these three teams. So it's not really a league... or maybe a league where you play unknown number of times with others.
As I said I experimented for maybe 5 minutes with your software and I felt it just can't handle it.
But if you tell me that I'm wrong I can give it another chance ;)

**Comment by robigan** [comment]
Same here, some better docs regarding usage would be much appreciated, we're using bracket to handle some small esports tournaments, and we've been slowly figuring stuff out. I'd be down to write some docs if need be

**Comment by evroon** [comment]
Yes I agree some usage documentation is missing, I don't think I will have time to work on it in the short term. PRs are greatly appreciated in any case! Especially for things like this because someone else with a fresh perspective can more clearly document how bracket should be used than me.

**Comment by evroon** [comment]
As proposed in https://github.com/evroon/bracket/discussions/1100, a terminology list would be nice as well
